content/en/docs/contribute/blog/guidelines.md

```diff
@@ -0,0 +1,227 @@
+* [Post-release announcements and updates](#post-release-comms)
+* Messages about important Kubernetes security vulnerabilities
+* Kubernetes projects updates
+* Tutorials and walkthroughs
+* Thought leadership around Kubernetes and cloud native
+* The components of Kubernetes are purposely modular, so writing about existing integration
+  points like CNI and CSI are on topic. Provided you don't write a vendor pitch, you can also write
+  about what is on the other end of these integrations.
+
+
+Here are some examples of content that is appropriate for the Kubernetes contributor blog:
+
+* Articles about how to test your change to Kubernetes code
+* Content around non-code contribution
+* Discussions about alpha features where the design is still under discussion
+* "Meet the team" articles about working groups, special interest groups etc
+* A guide about how to write secure code that will become part of Kubernetes itself
+* Articles about maintainer summits and the outcome of those summits
+
+### Examples of content that wouldn't be accepted {#what-we-do-not-publish}
+
+However, the project usually doesn't publish:
+
+* vendor pitches
+* an article you've published elsewhere, even if only to your own low-traffic blog
+* updates about an external project that works with our relies on Kubernetes (put those on
+  the external project's own blog)
+* articles about using Kubernetes with a specific cloud provider
+* articles that criticise specific people, groups of people, or businesses
+* articles that have important technical mistakes or misleading details (for example: if you
+  recommend turning off an important security control in production clusters, because it can
+  be inconvenient, the Kubernetes project is likely to reject the article).
+
+### Other considerations {#content-considerations-other}
+
+The Kubernetes website has an ICP licence from the government of China. It's unlikely to be a problem but we also cannot publish articles that would be blocked by the Chinese government's official filtering of internet content.
+
+Topics related to participation in or results of Kubernetes SIGs activities are always on
+topic (see the work in the [Contributor Comms Team](https://github.com/kubernetes/community/blob/master/communication/contributor-comms/blogging-resources/blog-guidelines.md#contributor-comms-blog-guidelines)
+for support on these posts).
+
+The project typically mirrors these articles to both blogs.
+
+
+- Blog posts should be original content
+
+  - The official blog is not for repurposing existing content from a third party as new content.
+  - The [license](https://github.com/kubernetes/website/blob/main/LICENSE) for the blog allows
+    commercial use of the content for commercial purposes, but not the other way around.
+
+- Blog posts should aim to be future proof
+
+  - Given the development velocity of the project, we want evergreen content that won't require
+    updates to stay accurate for the reader. 
```

Comment:
Because this is for a blog, I thought we wouldn't need to always consider how evergreen it should be? (And there is a separate frontmatter support for the evergreen articles)

Reply:
```suggestion
  - Given the development velocity of the project, we prefer timeless content that won't require
    updates to stay accurate for the reader.
```